src/selectors/columnWidths.js

```diff
@@ -41,7 +41,7 @@ let columnDefinition;
  * }} The total width of all columns.
  */
 function columnWidths(columnGroupProps, columnProps, scrollEnabledY, width) {
-  const scrollbarSpace = scrollEnabledY ? Scrollbar.SIZE + Scrollbar.OFFSET : 0;
+  const scrollbarSpace = scrollEnabledY ? SCROLLBAR_STYLE.size : 0;
```

Comment:
Not really sure how the existing logic worked, but if 
`Scrollbar.SIZE + Scrollbar.OFFSET === SCROLLBAR_STYLE.size` in this line,
then wouldn't the lines below where 
`Scrollbar.SIZE` is just replaced with `SCROLLBAR_STYLE.size` without `taking Scrollbar.OFFSET` into account be wrong?

So can you check if the calculations here and below are not wrong (even by 1 pixel, which is the value for Scrollbar.OFFSET)

Reply:
because I haven't included `offset` even in code, so test cases passing, I will restore that one pixel once you confirm if it is necessary with updated scroll bars